proto/minder/v1/minder.proto

```diff
@@ -2559,70 +2559,73 @@ message ListEvaluationHistoryResponse {
 }
 
 message EvaluationHistory {
-    // entity contains details of the entity which was evaluated.
-    EvaluationHistoryEntity entity = 1;
+    message Entity {
+        // id is the ID of the entity.
+        string id = 1;
 
-    // rule contains details of the rule which the entity was evaluated against.
-    EvaluationHistoryRule rule = 2;
+        // type is the entity type.
+        minder.v1.Entity type = 2;
 
-    // status contains the evaluation status.
-    EvaluationHistoryStatus status = 3;
+        // name is the entity name.
+        string name = 3;
+    }
 
-    // alert contains details of the alerts for this evaluation.
-    EvaluationHistoryAlert alert = 4;
+    message Rule {
+        // name is the name of the rule instance.
+        string name = 1;
 
-    // remediation contains details of the remediation for this evaluation.
-    EvaluationHistoryRemediation remediation = 5;
-}
+        // type is the name of the rule type.
+        string type = 2;
 
-message EvaluationHistoryEntity {
-    // id is the ID of the entity.
-    string id = 1;
+        // profile is the name of the profile which contains the rule.
+        string profile = 3;
+    }
 
-    // type is the entity type.
-    Entity type = 2;
+    message Status {
+        // status is one of (success, error, failure, skipped)
+        // not using enums to mirror the behaviour of the existing API contracts.
+        string status = 1;
 
-    // name is the entity name.
-    string name = 3;
-}
+        // details contains optional details about the evaluation.
+        // the structure and contents are rule type specific, and are subject to change.
+        string details = 2;
 
-message EvaluationHistoryRule {
-    // name is the name of the rule instance.
-    string name = 1;
+        // created_at is the timestamp of creation of this evaluation
+        google.protobuf.Timestamp evaluated_at = 3;
+    }
 
-    // type is the name of the rule type.
-    string type = 2;
+    message Remediation {
+        // status is one of (success, error, failure, skipped, not available)
+        // not using enums to mirror the behaviour of the existing API contracts.
+        string status = 1;
 
-    // profile is the name of the profile which contains the rule.
-    string profile = 3;
-}
+        // details contains optional details about the remediation.
+        // the structure and contents are remediation specific, and are subject to change.
+        string details = 2;
```

Comment:
Ditto on stringly-typed `details`.  Again, I'm willing to accept this as a _bridge_, but we should get to a more well-defined outcome.

Reply:
This was discussed during the design review - the proposal was to match the current state of Minder and work on improving remediations/alerts later.